### ISSUE TITLE
Can't mount both jump jets and mechanical boosters on the same 'Mech.

### ISSUE THREAD

**Issue by neoancient** [MegaMek/megameklab#44]
Per TacOps, p. 293, the two systems can be mounted on the same unit, but cannot both be used in the same turn. MML only allows one type of jumping system.

**Comment by arlith** [comment]
This is also reported on the MM SF tracker: https://sourceforge.net/p/megamek/bugs/3792/.  This largely ends up being more of a MM issue than a MML issue, because `Entity` needs a way to track which jump system is being used.

I've avoided this for a while, but it may not really be all that difficult.  Essentially, all we need is to keep track of whether the two systems exist and if they do, which one is currently being used.  Then, we need a way within MM to switch between the two systems.  This could probably be handled in the UnitDisplay > Systems tab.

While the construction rules allow this, I don't think any canon designs make use of it.

**Comment by HammerGS** [comment]
I don't think there are any mechs but I think there might be a couple of BA, if that is relevant.  But I'm not near MM to check.

**Comment by arlith** [comment]
I don't believe it matters for BA, as jumping works slightly differently for BA.  The mechanical jump boosters for BA I think just bolster their normal jumping movement.  For 'mechs, there are rule differences (ie, with mechanical jump boosters, 'mechs can only jump in a straight line).
